helm/superset/templates/deployment-beat.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.supersetBeat.enabled -}}
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "superset.fullname" . }}-beat
```

Comment:
I initially named the pods the same way you did, but some colleague said that `beat` is kind of generic, and it would be better naming it `{{ template "superset.fullname" . }}-celerybeat`.
I let you judge :wink:

Reply:
Yeah that makes sense...